model/event/task_event.go

```diff
@@ -173,11 +173,6 @@ func LogManyTaskAbortRequests(taskIds []string, userId string) {
 		TaskEventData{UserId: userId})
 }
 
-func LogTaskScheduled(taskId string, execution int, scheduledTime time.Time) {
```

Comment:
Right now the UI resolver looks for scheduled events to avoid cluttering up the UI:
https://github.com/evergreen-ci/evergreen/blob/a1df2bd607681cc3c5f60de215abce9c855a82dd/graphql/resolvers.go#L2048
Should we make a note to remove that logic after the TTL?

Reply:
I added TODOs and opened [EVG-16969](https://jira.mongodb.org/browse/EVG-16969)